monkey/infection_monkey/i_puppet.py

```diff
@@ -10,7 +10,9 @@ class PortStatus(Enum):
     CLOSED = 2
 
 
+ExploiterResultData = namedtuple("ExploiterResultData", ["result", "info", "attempts"])
 PortScanData = namedtuple("PortScanData", ["port", "status", "banner", "service"])
+PostBreachData = namedtuple("PostBreachData", ["command", "result"])
```

Comment:
These should probably be defined elsewhere as dataclasses(strings are more prone to errors and harder to debug). I'm here to read the `IPuppet` interface, not the data structures of all the modules. Now I'm forced to read through them before I get to the actual interface

Reply:
They're placeholders for the moment. As we implement the actual functionality, we'll work through these.